{"name": "test_19_phone_field_invalid", "status": "passed", "description": "test that checks if the user can not login to site with invalid phone number", "start": 1680585089106, "stop": 1680585105051, "uuid": "20ea7ef0-2531-4ec6-b2af-e578e72ca47f", "historyId": "65f16af3c0542432eda4da7f03e4fe4c", "testCaseId": "4c127932c49e62eac4ab2574608b775b", "fullName": "tests.test_login.TestLogin#test_19_phone_field_invalid", "labels": [{"name": "severity", "value": "Critical, sanity"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "AlexanderZak"}, {"name": "thread", "value": "4068-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}